rudder-agent/SOURCES/patches/fusioninventory/20845-yum-update-add-fields.patch

```diff
@@ -0,0 +1,153 @@
+diff --git a/lib/FusionInventory/Agent/Inventory.pm b/lib/FusionInventory/Agent/Inventory.pm
+index 2a7cbb4..6dbf05c 100644
+--- a/lib/FusionInventory/Agent/Inventory.pm
++++ b/lib/FusionInventory/Agent/Inventory.pm
+@@ -87,7 +87,7 @@ my %fields = (
+                             UNINSTALL_STRING URL_INFO_ABOUT VERSION
+                             VERSION_MINOR VERSION_MAJOR GUID ARCH USERNAME
+                             USERID SYSTEM_CATEGORY SOURCE_NAME SOURCE_VERSION/ ],
+-    SOFTWAREUPDATES  => [ qw/NAME VERSION ARCH FROM SOURCE KIND/ ],
++    SOFTWAREUPDATES  => [ qw/NAME VERSION ARCH FROM SOURCE KIND SEVERITY DESCRIPTION ID/ ],
```

Comment:
i think that if you want several ID in the SOFTWAREUPDATES you need  to create a <CVES><CVE></CVE></CVES> , but that would be clunky